cygnus-common/src/main/java/com/telefonica/iot/cygnus/utils/CommonUtils.java

```diff
@@ -32,7 +32,9 @@
 import java.util.GregorianCalendar;
 import java.util.Properties;
 import java.util.TimeZone;
+import java.util.UUID;
 import java.util.regex.Pattern;
+import static org.eclipse.jetty.ajp.Ajp13PacketMethods.LOCK;
```

Comment:
This import was necessary when `generateUniqueId` method was part of `NGSIUtils.java`. Unnecessary now.